source/_integrations/nest.markdown

```diff
@@ -66,16 +66,38 @@ For the first phase, you will turn on the API and create the necessary credentia
 1. Now the "Device Access Console" should be visible. Click on **Create project**.
     ![Screenshot of creating a project](/images/integrations/nest/create_project.png)
 
-1. Give your project a name and click **Next**.
+1. Give your Device Access project a name and click **Next**.
     ![Screenshot of naming a project](/images/integrations/nest/project_name.png)
 
-1. Next you will be asked for an *OAuth client ID*. It is a good idea to go create that now. Open a new tab to the [Google Cloud Console](https://console.developers.google.com/apis/credentials) **Credentials** page and click **Create
-Credentials**.
+1. Next you will be asked for an *OAuth client ID*. It is a good idea to go create that now.  Open a new tab to the [Google API Console](https://console.developers.google.com/apis/credentials)
+
+1. If this is your first time here, you likely need to create a new Google API project.  Click **Create Project** then **New
+Project**.  Note: This is different type of project from the Device Access project you are also creating.
+    ![Screenshot of APIs and Services Cloud Console with no existing project](/images/integrations/nest/api_project_needed.png)
+
+1. Give your API Project a name then click **Create**.  Note: You can ignore the *Project ID* here as Home Assistant does not need it.
+
+1. Click *OAuth consent screen* and make sure you have that configured, otherwise you can do that now.
+    ![Screenshot of OAuth consent screen creation](/images/integrations/nest/oauth_consent_create.png)
```

Comment:
```suggestion
1. Click *OAuth consent screen* and make sure you have that configured, otherwise, you can do that now.
    ![Screenshot of OAuth consent screen creation](/images/integrations/nest/oauth_consent_create.png)
```